tools/asset-sync/assets.ps1

```diff
@@ -0,0 +1,521 @@
+Set-StrictMode -Version 3
+
+# See https://stackoverflow.com/a/14440066 for a succinct explanation. We will be using
+# this methodology for any global variables that can be considered constants
+. (Join-Path $PSScriptRoot ".." ".." "eng" "common" "scripts" "common.ps1")
+
+$REPO_ROOT = Resolve-Path (Join-Path $PSScriptRoot ".." "..")
+$ASSETS_STORE = (Join-Path $REPO_ROOT ".assets")
```

Comment:
I'm not even sure I like this suggestion because it seems like a better convention to have dot sourcing imports above constant declarations:

```suggestion
$REPO_ROOT = Resolve-Path (Join-Path $PSScriptRoot ".." "..")
$ASSETS_STORE = (Join-Path $REPO_ROOT ".assets")

. (Join-Path $REPO_ROOT "eng" "common" "scripts" "common.ps1")
```

Reply:
Even this looks odd.  All of the OSes we use support forward slashes in paths. Is there a reason we don't standardize on:

``` ps
$REPO_ROOT = Resolve-Path "$PSScriptRoot/../.."
$ASSETS_STORE = "$REPO_ROOT/.assets"
```